lib/jsTask.js

```diff
@@ -0,0 +1,59 @@
+// task1
+const a1 = [1, 3, 5, 2, 4];
+const result = a1.filter((element, index) => index % 2 === 0);
+console.log(result);
+// O(n)
+
+
+
+
+// task2
+const a2 = [1, 3, 5, 2, 4];
+const result2 = a2.map(element => element ** 2);
+console.log(result2)
+// O(n)
+
+
+
+
+// task3
+const a3 = [1, 3, 5, 2, 4];
+// Generate the result3 array by combining the squared odd and even elements
+const result3 = [
+    // Use filter and map to handle odd elements
+    // Filter odd elements (element % 2 !== 0)
+    // Then, map each odd element to its square (element ** 2)
+    ...a3.filter(element => element % 2 !== 0)
+        .map(element => element ** 2),
+
+    // Use filter and map to handle even elements
+    // Filter even elements (element % 2 === 0)
+    // Then, map each even element to its square (element ** 2)
+    ...a3.filter(element => element % 2 === 0)
+        .map(element => element ** 2)
+];
+
+console.log(result3); // Output: [1, 25, 16] 
```

Comment:
The output is something else altogether.
![Screenshot 2024-04-29 at 17 29 42](https://github.com/krishnakalyana/coralmango-t1/assets/186976/1624bcc0-cbff-4cc6-a087-6a620aa98838)

Please fix